app/controllers/application_controller.rb

```diff
@@ -42,6 +42,32 @@ def sanitize_ac_params
     end
   end
 
+  include Pagy::Backend
+  def pagy(collection, **vars)
+    pagy_overflow_handler do
+      super
+    end
+  end
+
+  def pagy_query_result(query_result, **vars)
+    pagy_overflow_handler do
+      Pagy.new(
+        count: query_result.total_entries,
+        page: query_result.current_page,
+        limit: query_result.per_page,
+        **vars
+      )
+    end
+  end
+
+  def pagy_overflow_handler(&block)
+    begin
+      yield
+    rescue Pagy::OverflowError
+      nil
+    end
+  end
```

Comment:
Keeping the same behavior as now by not showing the pagination in case of overflow ("page 66 of 10").

pagy's [alternative options](https://ddnexus.github.io/pagy/docs/extras/overflow/) for reference.

Reply:
Without the `require 'pagy/extras/overflow'` line, Pagy always raises an exception in case of overflow ([source](https://ddnexus.github.io/pagy/docs/how-to/#handle-pagyoverflowerror-exceptions))